admin_ui/backend/agents_migration.py

```
@@ -68,6 +85,25 @@ def merged_effective_contexts(yaml_path: str, contexts_dir: str) -> dict:
+        try:
+            local_doc = yaml.safe_load(open(local_yaml_path)) or {}
+            local_inline = local_doc.get("contexts") or {} if isinstance(local_doc, dict) else {}
+            if isinstance(local_inline, dict):
+                merged = _deep_merge_dicts(merged, local_inline)
+                for key in local_inline:
+                    if key in merged and isinstance(merged[key], dict):
+                        merged[key]["_source_file"] = os.path.basename(local_yaml_path)
+        except Exception:
+            # Match runtime behavior: a broken optional override does not take
+            # down base configuration or migration status.
+            pass
```

Ruff (0.15.20)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)